exporter/prometheus/prometheus_test.go

```diff
@@ -343,6 +343,93 @@ func TestCumulativenessFromHistograms(t *testing.T) {
 	}
 }
 
+func TestHistogramUnorderedBucketBounds(t *testing.T) {
+	exporter, err := NewExporter(Options{})
+	if err != nil {
+		t.Fatalf("failed to create prometheus exporter: %v", err)
+	}
+	view.RegisterExporter(exporter)
+	reportPeriod := time.Millisecond
+	view.SetReportingPeriod(reportPeriod)
+
+	m := stats.Float64("tests/bills", "payments by denomination", stats.UnitDimensionless)
+	v := &view.View{
+		Name:        "cash/register",
+		Description: "this is a test",
+		Measure:     m,
+
+		// Intentionally used unordered elements in the ascending distribution
+		// to ensure unordered bucket bounds are handled.
+		Aggregation: view.Distribution(10, 5, 1, 50, 20, 100, 250),
```

Comment:
Also make sure that we duplicate a bucket boundary in there as the previous code also deduplicated boundaries :)

Reply:
SG, done.